pkg/diagnostics/systemd/systemd.go

```diff
@@ -108,7 +108,7 @@ log message:
   component. Check pod logs and recreate it with the correct CA cert.
   Routers and registries won't work properly with the wrong CA.
 * If it is from a node IP, the client is likely a node. Check the
-  openshift-node logs and reconfigure with the correct CA cert.
+  atomic-openshift-node logs and reconfigure with the correct CA cert.
```

Comment:
"Node logs" would generalize nicely to be used for both. But I guess I'm inclined to try to be specific for sysadmins who might not immediately mentally map that phrase to specific unit logs.